src/arrayReverse.test.js

```diff
@@ -8,13 +8,41 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(arrayReverse([])).toBeInstanceOf(Array);
+  });
 
+  it(`should return an reversed array
+    if original array consists array`, () => {
+    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse(['', ''])).toEqual(['', '']);
   });
 
-  // write more tests here
+  it(`should return an reversed array
+    if original array is palindrom`, () => {
+    expect(arrayReverse(['deed', 'peep'])).toEqual(['peep', 'deed']);
+  });
+
+  it(`should return an empty string
+    if original array is numberic`, () => {
+    expect(arrayReverse([1234, 5678])).toEqual(['', '']);
```

Comment:
no need to check it, the function accepts only an array of strings

Reply:
Done)

Reply:
i guess you forgot to commit and push your changes